src/System.Management.Automation/resources/ParameterBinderStrings.resx

```diff
@@ -142,7 +142,7 @@
     <value>Cannot bind positional parameters because no names were given.</value>
   </data>
   <data name="AmbiguousParameterSet" xml:space="preserve">
-    <value>Parameter set cannot be resolved using the specified named parameters.</value>
+    <value>Parameter set cannot be resolved using the specified named parameters.One or few of the parameters mentioned cannot be used together.Do a Get-command {0} to see the available parameter sets.</value>
```

Comment:
* Add spaces between periods and the first word of a sentence
* `One or few` should be `One or more` but, this could also occur when an insufficient number of parameters have been supplied, not just when 2 parameters cannot be used together. 
* `parameters mentioned` should be `parameters issued`.
* `Do a Get-command {0}` do all instances of this error pass the current command name to this string for formatting? if not, this could be very confusing.
* `Get-Help` would be preferable to `Get-Command` as `Get-Command` does not show this information be default.
*  `parameter sets` is some what esoteric to function./cmdlet authors. To an admin user it might be better to say `For syntax help`.

Reply:
Hi , Thanks for the review, I'll do the required modifications